frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx

```diff
@@ -0,0 +1,819 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import {
+  Button,
+  createStyles,
+  FormControl,
+  FormControlLabel,
+  InputAdornment,
+  Input,
+  LinearProgress,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+  withStyles,
+  WithStyles,
+  IconButton,
+} from '@material-ui/core';
+import {
+  BarChartOutlined,
+  DateRangeRounded,
+  CloseRounded,
+} from '@material-ui/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import DatePicker from 'react-datepicker';
+import { isNil, range } from 'lodash';
+import moment from 'moment';
+import {
+  mapSelector,
+  layersSelector,
+  layerDataSelector,
+} from '../../../../context/mapStateSlice/selectors';
+import { useUrlHistory } from '../../../../utils/url-utils';
+import { availableDatesSelector } from '../../../../context/serverStateSlice';
+import {
+  AnalysisDispatchParams,
+  PolygonAnalysisDispatchParams,
+  analysisResultSelector,
+  clearAnalysisResult,
+  isAnalysisLoadingSelector,
+  requestAndStoreAnalysis,
+  requestAndStorePolygonAnalysis,
+  setIsMapLayerActive,
+} from '../../../../context/analysisResultStateSlice';
+import {
+  AdminLevelType,
+  AggregationOperations,
+  AdminLevelDataLayerProps,
+  HazardDataType,
+  RasterType,
+  WMSLayerProps,
+  LayerKey,
+  BoundaryLayerProps,
+  GeometryType,
+} from '../../../../config/types';
+import {
+  getAdminLevelCount,
+  getAdminLevelLayer,
+} from '../../../../utils/admin-utils';
+import { LayerData } from '../../../../context/layers/layer-data';
+import {
+  LayerDefinitions,
+  getDisplayBoundaryLayers,
+} from '../../../../config/utils';
+import { useSafeTranslation } from '../../../../i18n';
+import { getDateFromList } from '../../../../utils/data-utils';
+import { getPossibleDatesForLayer } from '../../../../utils/server-utils';
+import {
+  downloadCSVFromTableData,
+  BaselineLayerResult,
+  ExposedPopulationResult,
+  PolygonAnalysisResult,
+  useAnalysisTableColumns,
+} from '../../../../utils/analysis-utils';
+import {
+  refreshBoundaries,
+  safeDispatchRemoveLayer,
+  safeDispatchAddLayer,
+} from '../../../../utils/map-utils';
+import { removeLayer } from '../../../../context/mapStateSlice';
+import { DEFAULT_DATE_FORMAT } from '../../../../utils/name-utils';
+import LayerDropdown from '../../Layers/LayerDropdown';
+import SimpleDropdown from '../../../Common/SimpleDropdown';
+import AnalysisTable from '../../Analyser/AnalysisTable';
+import { Extent } from '../../Layers/raster-utils';
+
+function AnalysisPanel({
```

Comment:
Can you delete the equivalent code in Analyser/index.tsx that is not used anymore?